model/team.go

```diff
@@ -19,7 +19,7 @@ type Team struct {
 	UpdateAt       int64  `json:"update_at"`
 	DeleteAt       int64  `json:"delete_at"`
 	Name           string `json:"name"`
-	Domain         string `json:"domain"`
+	URLId          string `json:"urlid"`
```

Comment:
It's kind of the team identifier but I guess we can't call it that, maybe TeamHandle?